corehq/messaging/scheduling/view_helpers.py

```diff
@@ -218,11 +218,15 @@ def update_rule_messages(self, rule, messages):
 
     def save_immediate_schedule(self, schedule, messages):
         AlertSchedule.assert_is(schedule)
+        assert(len(messages) == 1)
```

Comment:
debugging this will be easier if you provide some context to be displayed if the assertion fails. For example:
```py
assert len(messages) == 1, messages
```
(the same applies to assertions below)